t/router/radixtree-sni.t

```diff
@@ -951,7 +951,7 @@ connected: 1
 failed to do SSL handshake: certificate host mismatch
 --- error_log
 lua ssl server name: "aa.bb.test2.com"
-failed to found any SSL certificate by SNI: aa.bb.test2.com matched SNIs: ["moc.2tset","moc.2tset.*"]
+failed to find any SSL certificate by SNI: aa.bb.test2.com matched SNIs: ["moc.2tset","moc.2tset.*"]
```

Comment:
Is there a way to print the correct SNI value?

`["moc.2tset","moc.2tset.*"]` was wrong, it was reversed, it is not easy to read for us.